README.md

```diff
@@ -65,8 +66,10 @@ errbot -T
 Execute to configure the bot:
 
 ```
-!plugin config Stash
-{'STASH_URL': 'https://eden.esss.com.br/stash',
+!plugin config Stash {
+'STASH_URL': 'https://eden.esss.com.br/stash',
+'STASH_PROJECTS': ['ESSS'],
+'GITHUB_ORGANIZATIONS': ['ESSS'],
```

Comment:
Let's add a new configuration option: `GITHUB_INTEGRATION_ENABLED`, defaulting to `True`, just in case we encounter a blocking bug we can disable the github support without having to revert and redeploy like next time.

Reply:
Oops, I implemented before reading this comment, so I called it `ENABLE_GITHUB` and it defaults to `False`. 😧 
Let me know if I should change it.